README.md

```diff
@@ -1,43 +1,32 @@
-# Foundation Bootstrap Team
+# The OpenJS Foundation Cross Project Council
 
-This is the operating repository for the Foundation Bootstrap Team.
+The OpenJS Foundation Cross Project Council (CPC) is the technical governing body of the OpenJS Foundation. It is described in the [CPC Charter](./CPC-CHARTER.md)
 
-The Foundation Bootstrap Team is the temporary body tasked with managing community recommendations for the transition of the merged Node.js and JS Foundation to a permanent governance model.
+## CPC Scope
 
-The files in this repository are living documents and will never represent finished governance. Once we reach consensus on governance, this repository will be archived and the documents will be moved to their new home in a Foundation specific Github org.
+**The CPC is responsible for ensuring collaboration is the driving principle within a Project, between OpenJS Foundation Projects, and between OpenJS Foundation Projects and the broader community.**
 
-Until such a time that this group has consensus on a new governance model, and the Foundation has any legal requirements satisfied, all projects, programs, and committees will continue to operate unchanged.
+The CPC exercises autonomy in managing its responsibilities and seeks agreement from the OpenJS Foundation Board on any change to the scope of those responsibilities.
 
-Upon enactment of this new governance model, this working body will be disbanded.
+### OpenJS Fondation Projects
 
-> Note: If there is to be an extended Bootstrap transition committee to, for example, provide oversight in the organization's first year, those criteria must be defined as a new program in the proposed governance model. There is no extension of **this** specific group's life-cycle.
+* To be filled in
 
+## OpenJS Foundation Members
 
-## Definitions
-Throughout this repository we may use assorted acronyms and jargon to describe many complex concepts. This language is subject to change as we iterate on proposals. To help onboard and level-set, we will maintain an active list of commonly used language here. If you see domain specific language used in this repository that is not represented here, please add it!
+### Voting Memberss
 
```

Comment:
* @mcollina - Matteo Collina <matteo.collina@gmail.com> (he/him)

Reply:
add
```suggestion
- **WebdriverIO**: Christian Bromann (@bromann)
```

Reply:
@MylesBorins could you clarify the intent/reason of including this list? Given the context of it as you've written compared to the comments on it, I'm not sure if I'm missing something.